_docs/usage.md

```diff
@@ -138,6 +140,19 @@ thelounge install thelounge-plugin-closepms
 
 The Lounge automatically loads the plugin after installation and activates it.
 
+For local plugins, see [Installing local packages](#installing-local-packages)
+
+## Installing local packages
+
+During development of packages or if uploading a package to the npm registry is undesired, packages can be installed from the local file system.
+
+That's done by adding a file: prefix to package name:
+
+```
+thelounge install file:~/path/to/package_dir
```

Comment:
Note: this won't actually work until https://github.com/thelounge/thelounge/pull/4610 is merged